pages/project-version/update/submit/schema/index.js

```diff
@@ -13,7 +13,7 @@ const getDateField = establishmentName => {
   };
 };
 
-const getAwerbQuestion = ({ isLegacy, isAmendment, awerbEstablishments }) => {
+const getAwerbQuestion = ({ isLegacy, canBeAwerbExempt, awerbEstablishments }) => {
```

Comment:
`isAmendment` previously included transfers in certain circumstances, but transfers cannot be exempt.